Cargo.toml

```
@@ -16,7 +16,7 @@ rust-version = "1.56"
 libc = { version = "0.2", default-features = false }
 
 [target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.52"
+version = "0.59"
```

@lambda-fairy would a change like one of the following work for you:
```suggestion
version = ">=0.52, <0.60"
```
or 
```suggestion
version = ">=0.52, <=0.59"
```
If there's no actual breaking changes to any of the apis you're using, wouldn't this allow users to use a wider range of versions of `windows-sys` without needing to bump the msrv? This would also help prevent downstream triggers of `clippy::multiple-crate-versions` (ie multiple versions of `windows-sys` getting pulled in and compiled) due to some crates wanting newer versions and some crates wanting older ones.

There seems to be precedent for doing this is various popular crates:
* https://lib.rs/crates/windows-targets/rev
* https://lib.rs/crates/windows-sys/rev
* https://lib.rs/crates/windows/rev